redbeams/src/main/java/com/sequenceiq/redbeams/converter/spi/ServiceEndpointCreationToEndpointTypeConverter.java

```diff
@@ -0,0 +1,24 @@
+package com.sequenceiq.redbeams.converter.spi;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.common.model.EndpointType;
+import com.sequenceiq.environment.api.v1.environment.model.base.ServiceEndpointCreation;
+
+@Component
+public class ServiceEndpointCreationToEndpointTypeConverter {
```

Comment:
this is almost duplicate class, but it slightly different from the one in env service. Is it on purpose?